OCS-CI's HAProxy is misconfigured (SSL_ERROR_RX_RECORD_TOO_LONG / ERR_SSL_PROTOCOL_ERROR)

https://ocs4-jenkins-csb-ocsqe.apps.ocp4.prod.psi.redhat.com/job/qe-deploy-ocs-cluster/2212/

same issue for 
```
>       raise exception_class(message, screen, stacktrace)
E       selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_SSL_PROTOCOL_ERROR
E         (Session info: headless chrome=90.0.4430.93)

venv/lib64/python3.8/site-packages/selenium/webdriver/remote/errorhandler.py:242: WebDriverException
```
https://ocs4-jenkins-csb-ocsqe.apps.ocp4.prod.psi.redhat.com/job/qe-deploy-ocs-cluster-prod/688/testReport/tests.manage.z_cluster.cluster_expansion.test_add_capacity/TestAddCapacity/test_add_capacity/


![image](https://user-images.githubusercontent.com/61982127/117298895-8e37ca80-ae80-11eb-983e-7462f48ee628.png)

--- a/ocs_ci/utility/load_balancer.py
+++ b/ocs_ci/utility/load_balancer.py
@@ -110,7 +110,7 @@ class LoadBalancer(object):
             for node in nodes:
                 cmd = (
                     f"sudo sed -i '0,/.*:{port} check$/s/.*:{port} check$/        server "
-                    f"{node} {node}:80 check\\n&/' {constants.HAPROXY_LOCATION}"
+                    f"{node} {node}:{port} check\\n&/' {constants.HAPROXY_LOCATION}"
                 )
                 self.lb.exec_cmd(cmd)
 

